commit 8b41273a6c470da18b0c49b2f8e68f4353ad133e
Author: Florian Weimer <fweimer@redhat.com>
Date:   Thu Nov 24 13:39:57 2022 +0100

    : hplip-pserror-c99.patch
    
    prnt/pserror.c: Replace an implicit int with an explicit int

diff --git a/prnt/hpps/pserror.c b/prnt/hpps/pserror.c
index 114d7e1b5fa364fb..493b49c27917a7e6 100644
--- a/prnt/hpps/pserror.c
+++ b/prnt/hpps/pserror.c
@@ -24,7 +24,7 @@ extern char *program ;	/* Defined by main program, giving program name */
 void message(int flags, char *format, ...)
 {
   va_list args ;
-  static column = 0 ;		/* current screen column for message wrap */
+  static int column = 0 ;	/* current screen column for message wrap */
   char msgbuf[MAX_MESSAGE] ;	/* buffer in which to put the message */
   char *bufptr = msgbuf ;	/* message buffer pointer */
 
